universal_transfer_operator/tests/conftest.py

```diff
@@ -91,10 +113,160 @@ def dataset_table_fixture(request):
         table.name = create_unique_str(UNIQUE_HASH_SIZE)
     file = params.get("file")
 
-    dp.populate_table_metadata(table)
+    dp.populate_metadata()
     # dp.create_schema_if_needed(table.metadata.schema)
 
     if file:
         dp.load_file_to_table(file, table)
     yield dp, table
     dp.drop_table(table)
+
+
+def set_table_missing_values(table: Table, dataset_name: str) -> Table:
+    """
+    Set missing values of table dataset
+    """
+    conn_id = DATASET_NAME_TO_CONN_ID[dataset_name]
+    table = table or Table(conn_id=conn_id)
+
+    if not table.conn_id:
+        table.conn_id = conn_id
+
+    if not table.name:
+        # We create a unique table name to make the name unique across runs
+        table.name = create_unique_str(UNIQUE_HASH_SIZE)
+
+    return table
+
+
+def set_file_missing_values(file: File, dataset_name: str):
+    """
+    Set missing values of file dataset
+    """
+    conn_id = DATASET_NAME_TO_CONN_ID[dataset_name]
+    if not file.conn_id:
+        file.conn_id = conn_id
+    return file
+
+
+def populate_file(src_file_path: str, dataset_provider: BaseFilesystemProviders, dp_name: str):
+    """
+    Populate file with local file data
```

Comment:
params are missing in docstrings here and subsequent methods

Reply:
I think it has enough context from the name and type in function definition, do you still feel we should add it?

Reply:
Done.